src/components/AppsMenu/AppsMenuContent.jsx

```diff
@@ -36,13 +35,15 @@ export const AppsMenuContent = ({ isFetchingApps, apps, homeApp }) => {
 
   const homeSlug = homeApp && homeApp.slug
 
-  const displayedApps = apps
-    .filter(app => app.slug !== homeSlug)
-    .sort(sorter(getAppDisplayName))
+  const displayedApps = apps.filter(app => app.slug !== homeSlug)
+
+  const sortedApps = flag('apps.sort')
```

Comment:
maybe we should export it to use same method in here and Home 🤔

Reply:
`sortApplicationsList` comes from cozy-client applications model. It is enough no ?